src/AppComponent.js

```diff
@@ -0,0 +1,23 @@
+import React, {Component} from 'react';
+import styled from 'styled-components';
+
+const StyledView = styled.p`
+  color: ${props => props.color};
+`;
+
+class AppComponent extends Component {
+
+    render() {
+        const color = this.props.color;
```

Comment:
ลองใช้แบบนี้ดูนะ 
`const { color, handleChange } = this.props;`
 
จะได้สั้นๆครับผม